.github/workflows/build-wasm_of_ocaml.yml

```diff
@@ -18,6 +18,8 @@ jobs:
         ocaml-compiler:
           - 4.14.x
           - 5.00.x
+          - 5.01.x
```

Comment:
I think we should remove this for now, and fix wasm_of_ocaml for OCaml 5.01 in another PR.